modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -61,7 +61,16 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 +
 [NOTE]
 ====
-If you do not control the DNS server, you can add the record to your `/etc/hosts` file. This action makes the API accessible to only you, which is not suitable for production deployment but does allow installation for development and testing.
+If you do not control the DNS server, you can access the cluster by adding the cluster domain names such as the following to your `/etc/hosts` file:
+
+* <API_FLOATING_IP> api.<CLUSTER_NAME>.<BASE_DOMAIN>
+* <APPLICATION_FLOATING_IP> `grafana-openshift-monitoring.apps.<CLUSTER_NAME>.<BASE_DOMAIN>`
+* <APPLICATION_FLOATING_IP> `prometheus-k8s-openshift-monitoring.apps.<CLUSTER_NAME>.<BASE_DOMAIN>`
+* <APPLICATION_FLOATING_IP> `oauth-openshift.apps.<CLUSTER_NAME>.<BASE_DOMAIN>`
+* <APPLICATION_FLOATING_IP> `console-openshift-console.apps.<CLUSTER_NAME>.<BASE_DOMAIN>`
+* <APPLICATION_FLOATING_IP> `integrated-oauth-server-openshift-authentication.apps.<CLUSTER_NAME>.<BASE_DOMAIN>`
```

Comment:
I'm confused about the way this information is presented. Is the user supposed to place the entire line "APPLICATION_FLOATING_IP> `grafana-openshift-monitoring.apps.<CLUSTER_NAME>.<BASE_DOMAIN>`" in the `/etc/hosts` file? If so, the entire line should be in backticks or the set of examples should be in a [codeblock](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). Also, the user-replaced items should be [lowercase](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values).

Reply:
Addressed